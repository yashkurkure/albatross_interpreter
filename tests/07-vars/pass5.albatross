var thisIsTheNameOfAVariable int := 1;
var anotherVariable string := "hello";
var anotherVariable char := 4 & 5;

return 6 ^ 7;
