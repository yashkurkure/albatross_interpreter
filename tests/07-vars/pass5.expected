(1)
("hello")
((4)&(5))
((6)^(7))

